manifest.json

```
@@ -1313,11 +1313,11 @@
       "mod_version": "3"
     },
     {
-      "description": "An integrated terminal for lite-xl. \\*([screenshot](https://user-images.githubusercontent.com/1034518/254662617-6b8003da-d4c1-4227-8fc9-3d2b1ae89bf2.png))",
-      "version": "0.9",
+      "description": "An integrated terminal for lite-xl. \\*([screenshot](https://github.com/adamharrison/lite-xl-terminal/assets/1034518/eb8a72a0-ff61-4b95-b009-364ac2725f70))",
```

Do we need that asterisk?
```suggestion
      "description": "An integrated terminal for lite-xl. ([screenshot](https://github.com/adamharrison/lite-xl-terminal/assets/1034518/eb8a72a0-ff61-4b95-b009-364ac2725f70))",
```

---

Was actually supposed to be italics; was following the example from `tabnumbers`, which also just had a star. Fixed.